Homework7-start/Homework7/User/SectionView.swift

```diff
@@ -33,11 +33,20 @@
 import SwiftUI
 
 struct SectionView: View {
```

Comment:
NICE! That's the idea of SwiftUI! Finding ways to extract views so that you can reuse them later.